drivers/sensor/tdk/icm456xx/icm456xx.c

```diff
@@ -0,0 +1,1077 @@
+/*
+ * Copyright (c) 2024 TDK Invensense
+ * Copyright (c) 2022 Esco Medical ApS
+ * Copyright (c) 2020 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/init.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/pm/device.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/drivers/sensor/icm456xx.h>
+#include <zephyr/drivers/sensor/tdk_apex.h>
+
+#include "icm456xx.h"
+#include "icm456xx_trigger.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(ICM456XX, CONFIG_SENSOR_LOG_LEVEL);
+
+/* Convert DT enum to sensor ODR selection */
+#define ICM456XX_CONVERT_ENUM_TO_ODR_POS (4)
```

Comment:
This define is a left-over from icm42x70 driver, it is not used in this file. Please remove it.

Reply:
Removed